lib/mbaasRequest/mbaasRequest.js

```diff
@@ -175,6 +175,15 @@ function responseHandler(params, cb) {
 
     if (err || (httpResponse && httpResponse.statusCode >= 400)) {
 
+      // MbaaS is unreachable
+      if (err.code !== undefined && err.code === "ENOTFOUND" || err.code === "ETIMEDOUT") {
```

Comment:
@nialldonnellyfh If i remove these checks, when should the "MBaaS environment is not reachable..." message be triggered to be shown? The err.code has the value ENOTFOUND/ETIMEDOUT when the mbaas is unreachable (httpResponse.statusCode is '200 ok' though..).